packages/account/package.json

```diff
@@ -35,7 +35,7 @@
         "@deriv-com/utils": "^0.0.25",
         "@deriv-com/ui": "1.29.3",
         "@deriv/api": "^1.0.0",
-        "@deriv-com/quill-ui": "1.13.17",
+        "@deriv-com/quill-ui": "1.13.20",
```

Comment:
Should we notify account team that I updated lib version?

Reply:
we can but in my opinion, patch release means there are no breaking changes. and btw, account team bumps the version up without notifying us :D